executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -313,6 +315,7 @@ impl<'a, T: FieldElement> MachineExtractor<'a, T> {
             }
             Identity::Polynomial(i) => self.fixed.polynomial_references(i),
             Identity::Connect(i) => self.fixed.polynomial_references(i),
+            Identity::PhantomBusInteraction(i) => self.fixed.polynomial_references(&i.tuple),
```

Comment:
Just double checking we only need to visit the tuple?

Reply:
Right, I should add the multiplicity! I thought it was handled elsewhere, but it is also added when processing lookups. Good catch!